src/classes/CalculatorClass.js

```diff
@@ -0,0 +1,52 @@
+export default class Calculator {
+  constructor(calculatedNumber, isValidNumber) {
+    this.calculatedNumber = calculatedNumber;
+    this.isValidNumber = isValidNumber;
+  }
```

Comment:
```suggestion
export default class Calculator {
  calculatedNumber; isValidNumber;
  constructor(calculatedNumber, isValidNumber) {
    this.calculatedNumber = calculatedNumber;
    this.isValidNumber = isValidNumber;
  }
```

멤버변수의 경우 이런식으로 클래스필드에 미리 정의해놓고 사용하면 다른 사람 입장에서 이 클래스에 어떤 멤버변수가 쓰이는지 알 수 있어서 편하답니다!

Reply:
오 이 부분은 몰랐던 방식이네요! 제가 생성자 함수 위에 미리 선언하는 것에 어색함을 느끼는 것 같아요 처음 봤을 때 제안 주신 방법이 오히려 헷갈릴 수 있겠다 라는 느낌을 받았거든요 🤔  좋은 조언 감사합니다 ㅎㅎ